testcontainer_test.go

```diff
@@ -64,7 +69,7 @@ func TestApp(t *testing.T) {
 	// Any requeset body will work since POST requests are not currently allowed
 	requestBody := []byte(`{"key": "value"}`)
 
-	resp, err = client.Post("http://" + host + ":" + strconv.Itoa(port.Int()) + "/", "application/json", bytes.NewBuffer(requestBody))
+	resp, err = client.Post("http://"+host+":"+strconv.Itoa(port.Int())+"/", "application/json", bytes.NewBuffer(requestBody))
```

Comment:
Losing the spaces around the `+` makes it harder to read, imho. And, different from other places (e.g., in main.go) where there is a space around them.